packages/framework/src/Model/Product/Search/Export/ProductSearchExportWithFilterRepository.php

```diff
@@ -123,6 +127,17 @@ public function getProductsDataForIds(int $domainId, string $locale, array $prod
         return $result;
     }
 
+    /**
+     * @param int $domainId
+     * @return int
+     */
+    public function getProductTotalCountForDomainAndLocale(int $domainId): int
```

Comment:
This method deserves to be renamed to `getProductTotalCountForDomain`, please don't forget upgrade notes

Reply:
what about this solution ? https://github.com/shopsys/shopsys/pull/1193/commits/c7924abe35b49590167e38467fedec56eac060a2